inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape.cpp

```diff
@@ -71,42 +70,42 @@ bool propagateUpperBoundFromExistingDSR(std::shared_ptr<ngraph::Function>& funct
-        {ngraph::opset3::Clamp::type_info,                     dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::Floor::type_info,                     dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::Log::type_info,                       dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::Relu::type_info,                      dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::ScatterUpdate::type_info,             dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::Sigmoid::type_info,                   dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::Softmax::type_info,                   dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::Exp::type_info,                       dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::Sqrt::type_info,                      dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::LogicalNot::type_info,                dynamicToStaticUnaryElementwise},
-        {ngraph::opset3::StridedSlice::type_info,              dynamicToStaticShapeStridedSlice},
-        {ngraph::opset3::Squeeze::type_info,                   dynamicToStaticShapeSqueeze},
-        {ngraph::opset3::Gather::type_info,                    dynamicToStaticShapeGather},
-        {ngraph::opset3::Unsqueeze::type_info,                 dynamicToStaticShapeUnsqueeze},
-        {ngraph::opset3::ROIAlign::type_info,                  dynamicToStaticShapeROIAlign},
-        {ngraph::opset3::Reshape::type_info,                   dynamicToStaticShapeReshape},
-        {ngraph::opset3::Broadcast::type_info,                 dynamicToStaticShapeBroadcast},
-        {ngraph::opset3::MatMul::type_info,                    dynamicToStaticShapeMatMul},
-        {ngraph::opset5::Split::type_info,                     dynamicToStaticShapeSplit},
+        {ngraph::opset3::Add::type_info,               dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset3::Multiply::type_info,          dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset3::Subtract::type_info,          dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset3::VariadicSplit::type_info,     dynamicToStaticShapeVariadicSplit},
+        {ngraph::opset3::Divide::type_info,            dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset3::Equal::type_info,             dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset3::Greater::type_info,           dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset3::Power::type_info,             dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset3::Maximum::type_info,           dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset3::Minimum::type_info,           dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset3::Less::type_info,              dynamicToStaticShapeBinaryEltwise},
+        {ngraph::opset5::NonMaxSuppression::type_info, dynamicToStaticNonMaxSuppression},
```

Comment:
It could be a tricky question, but do we need to updated all others keys to opest5 as well?

Reply:
Well, I don't think so. At some moment we have written and tested DTS's for opset3 versions of operations. The same operations in later opsets may be different, which means that the simple replace here can break the backward compatibility. In case we need to support some newer operation version from newer opset, we not need to replace, but to add a new operation here.

Reply:
Ok, thanks for explanation!